WooCommerce/WooCommerceTests/ViewRelated/Blaze/BlazeEditAdViewModelTests.swift

```diff
@@ -0,0 +1,187 @@
+import Foundation
+import XCTest
+import Yosemite
+import Photos
+@testable import WooCommerce
+
+final class BlazeEditAdViewModelTests: XCTestCase {
+    private let sampleAdData = BlazeEditAdData(image: MediaPickerImage(image: UIImage.emailImage,
+                                                                       source: .media(media: .fake())),
+                                               tagline: "Sample Tagline",
+                                               description: "Sample description")
+
+    // MARK: Tagline
+    func test_tagline_footer_text_is_plural_when_multiple_characters_remaining() {
+        // Given
+        let sut = BlazeEditAdViewModel(siteID: 123,
+                                       adData: sampleAdData,
+                                       onSave: { _ in })
+        // When
+        sut.tagline = sampleString(length: 1)
+
+        // Then
+        let expectedString = String(format: BlazeEditAdViewModel.Localization.LengthLimit.plural, 31)
+        XCTAssertEqual(sut.taglineFooterText, expectedString)
+    }
+
+    func test_tagline_footer_text_is_singular_when_one_character_remaining() {
+        // Given
+        let sut = BlazeEditAdViewModel(siteID: 123,
+                                       adData: sampleAdData,
+                                       onSave: { _ in })
+        // When
+        sut.tagline = sampleString(length: 31)
+
+        // Then
+        let expectedString = String(format: BlazeEditAdViewModel.Localization.LengthLimit.singular, 1)
+        XCTAssertEqual(sut.taglineFooterText, expectedString)
+    }
+
+    func test_tagline_footer_text_is_singular_when_zero_character_remaining() {
```

Comment:
this should be _plural_ instead of _singular_

Reply:
Good catch. Updated in https://github.com/woocommerce/woocommerce-ios/pull/11612/commits/b4bd01acad511c35068f8fb73c926329ec43a623